projects/sdk/src/lib/tokens.ts

```diff
@@ -319,9 +321,12 @@ export class Tokens {
 
     ////////// Groups //////////
 
+    const siloWhitelist = [this.BEAN, this.BEAN_CRV3_LP, this.UNRIPE_BEAN, this.UNRIPE_BEAN_CRV3];
+    this.siloWhitelist = new Set(siloWhitelist);
+    this.siloWhitelistAddresses = siloWhitelist.map((t) => t.address);
```

Comment:
We don't need this, an array of addresses can easily been derived from a Set of tokens where needed. 
`const addr = [...this.siloWhitelist].map(t => t.address)`

Reply:
Just now saw your other comment about this. You used it in two places, one of which didn't need to be an array. Not a big deal either way

Reply:
I'll take a look at this again and we can revert if it's not used very often